src/main/java/com/epam/ta/reportportal/entity/widget/content/NotPassedCasesContent.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.ta.reportportal.entity.widget.content;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * @author Ivan Budayeu
+ */
+public class NotPassedCasesContent extends AbstractLaunchStatisticsContent {
+
+	@JsonProperty(value = "% (Failed+Skipped)/Total")
```

Comment:
not sure it's a good name for json field

Reply:
It's like in the 4.3 version and our front-end team requested that name of field, cause it's more convinient for them